test/reconcile-prop-types/options.ts

```diff
@@ -0,0 +1,17 @@
+import { TestOptions } from '../types';
+
+const options: TestOptions = {
+	injector: {
+		removeExistingPropTypes: true,
+		reconcilePropTypes: (prop, previous: any, generated) => {
```

Comment:
I have no idea why this is rejected otherwise. Annotating it with `previous: string | undefined` doesn't help either.